students/ZachCooper/session05/mailroom3.py

```diff
@@ -0,0 +1,233 @@
+#!usr/bin/env python3
+
+from textwrap import dedent
+import math
+import os
+
+
+# Name letters will be under in disk
+OUT_PATH = "thank_you_letters"
+
+
+# Creates path to open cwd for letters
+def prepare_to_run():
+    if not os.path.isdir(OUT_PATH):
+        os.mkdir(OUT_PATH)
+
+
+# Donors db which is changed to a dict in main
+def disney_donor_db():
+    return {'yosemite sam': ("Yosemite Sam", [100.56, 200.23, 300]),
+            'yogi bear': ("Yogi Bear", [2000.33, 4000, 1000]),
+            'daffey duck': ("Daffey Duck", [5000.24, 10000.34, 30000]),
+            'wile e. coyote': ("Wile E. Coyote", [300.40, 1000, 750.17]),
+            'bugs bunny': ("Bugs Bunny", [100000.87, 50000, 100]),
+            'mickey mouse': ("Mickey Mouse", [1345.67, 345.78, 1.23]),
+            }
+
+
+# Runs main menu function
+def menu_selection(prompt, dispatch_dict):
+    # Really like the menu dict option using switches
+    while True:
+        response = input(prompt)
+        response = response[:1].lower()
+        if dispatch_dict.get(response, unknown)() == "exit menu":
+            break
+
+
+# First item function in menu
+def thank_you():
```

Comment:
could you use a dispatch dict here, too? with so few options, maybe not critical, but why not?